### DISCUSSION TITLE
Amazing plug-in

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana-infinity-datasource#313] [Show and tell]
Just want to ssy thanks very much for this incredible plug-in!!

One thing I want to mention is instead of csv would be nice to rename that to AnySV or Delimited cause this awesome tool can do any Delimiter

**Comment by yesoreyeram** [comment]
Thank you @yosiasz for the kind words.  I love the term AnySV 😄 but CSV/TSV makes sense for more people. If I ever rename, definitely will consider AnySV.
